Cabal/Distribution/Utils/Json.hs

```diff
@@ -1,7 +1,8 @@
--- | Utility json lib for Cabal
--- TODO: Remove it again.
-module Distribution.Simple.Utils.Json
+-- | Extremely simple JSON helper. Don't do anything too fancy with this!
```

Comment:
I think it was the idea to replace this with a dependency such as microaeson https://hackage.haskell.org/package/microaeson

Reply:
`Cabal` cannot take any dependencies easily.

Reply:
It looked like `Cabal` only depends on boot packages, hence why I thought it would just be easier to expose this module